examples/embed/example_voyageai.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from unstructured.documents.elements import Text
+from unstructured.embed.voyageai import VoyageAIEmbeddingConfig, VoyageAIEmbeddingEncoder
+
+# To use Voyage AI tou will need to pass Voyage AI API Key (obtained from https://dash.voyageai.com/)
```

Comment:
To use Voyage AI **you** will need to pass Voyage AI API Key (obtained from https://dash.voyageai.com/)

Reply:
Hehe, the funny thing is that it is not a misspell but a copy-paste problem. :-)